python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -108,22 +108,103 @@ def _apply_nms(orig_prediction: dict, iou_thresh: float = 0.5):
     return nms_prediction
 
 
-def _wrap_image_model(model, examples, model_task, is_function, number_of_classes=None):
+def _process_automl_detections_to_raw_detections(
+        image_detections,
+        label_dict: Dict[str, int],
+        image_size: Tuple[int, int]) -> Dict[str, Tensor]:
+    """Process AutoML mlflow object detections from a single image.
+
+    :param image_detections: AutoML mlflow detections
+    :type image_detections: list
+    :param label_dict: Label dictionary mapping class names to an index
+    :type label_dict: dict
+    :param image_size: Image size
+    :type image_size: tuple(int, int)
+    :return: Raw detections
+    :rtype: dict
+
+    The length of image_detections list will be equal to the number
+    of objects the AutoML MLflow model detected in a single image.
+    Below is an example of image_detections when the model detected
+    two objects:
+
+    [{'box': {'topX': 0.645,
+              'topY': 0.373,
+              'bottomX': 0.8276,
+              'bottomY': 0.6102
+                },
+      'label': 'can',
+      'score': 0.945},
+     {'box': {'topX': 0.324,
+              'topY': 0.5643,
+              'bottomX': 0.6511,
+              'bottomY': 0.865
+                },
+      'label': 'milk_bottle',
+      'score': 0.93}
+    ]
+    The bbox coordinates need to be scaled by the image dimensions.
+    """
+
+    x, y = image_size
+
+    boxes = []
+    scores = []
+    labels = []
+    for detection in image_detections:
+        # predicted class label
+        label = detection['label']
+        # confidence score from model
+        score = detection['score']
+        # predicted bbox coordinates
+        box = detection["box"]
+
+        ymin, xmin, ymax, xmax = (
```

Comment:
nit - I think topY = ymin, etc. here, which is a bit misleading. Shouldn't topy = ymax, and so on?

Reply:
Image coordinate conventions have the (0,0) to be the top-left corner. Therefore, ymax is actually lower on the image compared to ymin and so ymin is the "topY". Would a comment in the code help for clarification?